packages/repr/repr.0.1.0/opam

```diff
@@ -23,6 +23,7 @@ depends: [
   "hex" {with-test}
   "alcotest" {>= "1.1.0" & with-test}
   "odoc" {with-doc}
+  "ppx_repr" {= version & post & with-test}
```

Comment:
Looks like that by using post, it is installed after running the tests...
```suggestion
  "ppx_repr" {= version & with-test & post}
```
Do you think ordering may be an issue here? (I have no idea how post work)

Otherwise we could just disable the tests (or you could move these tests to `ppx_repr` instead)

Reply:
ah nah nevermind I hadn't realized this was a dependency from the same package. @CraigFe is there any way you could not use this dependency recursively? If the base library is not usuable without the ppx then the ppx must be merged in the main library.

Reply:
Hmm, `post` was not intended as a CI hack, and I think it should be possible to use this dependency set, since we were doing the same thing with `irmin` and `ppx_irmin` shortly before these packages were extracted to their own repository.

I'll take a look at it tomorrow.